Makefile

```diff
@@ -53,8 +53,8 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) crd:generateEmbeddedObjectMeta=true rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
is kustomize ```make install``` working after this change ?
can you confirm ?

Reply:
No this is what I wrote above regarding the `kubectl apply`.
`make manifests` runs `kubectl apply` and it won't work here because of the annotation.
Error: `metadata.annotations: Too long: must have at most 262144 bytes`
We need to decide if we want to try and remove the description, or try to bypass the `kubectl apply`

Reply:
we can add ` --server-side` for the `make manifest` in order to support the `kubectl apply` operation. Added it, to the relevant make commands and to the e2e tests. I think its better to have the descriptions in the CRD.